pkg/util/blockingcacheclient/client.go

```diff
@@ -186,7 +186,16 @@ type CacheStatusClient struct {
 	Cache *CacheClient
 }
 
-func (c *CacheStatusClient) Update(ctx context.Context, obj client.Object, opts ...client.UpdateOption) error {
+func (c *CacheStatusClient) Create(ctx context.Context, obj client.Object, subResource client.Object, opts ...client.SubResourceCreateOption) error {
```

Comment:
We need to implement the _Create_ method for the **CacheStatusClient** struct for it to comply to the SubResourceWriter interface. as per [alias](https://github.com/kubernetes-sigs/controller-runtime/blob/84c5c9f4e29bfd3cead5c89552c9bc7dc90c45ad/pkg/client/interfaces.go#L132) and [interface](https://github.com/kubernetes-sigs/controller-runtime/blob/84c5c9f4e29bfd3cead5c89552c9bc7dc90c45ad/pkg/client/interfaces.go#L140). 

Just wanted to know, whether it is valid here to have a _Create_ for Status subresource, as we usually avoid that and let the APIServer handle status subresource creation.

Reply:
I don't know if there are cases where this would actually get called, but since we need to fulfill the interface... :man_shrugging: